lib/cisco_node_utils/command_reference.rb

```diff
@@ -261,17 +263,17 @@ def debug(text)
       puts "DEBUG: #{text}" if @@debug
     end
 
-    KNOWN_APIS = %w(nxapi grpc)
+    KNOWN_FILTERS = %w(cli_nexus cli_ios_xr)
 
     # Helper method
     # Given a Hash of command reference data (as read from YAML), does:
-    # - Filter any API-specific data according to the given api string
-    # - Filter any platform-specific data according to the given product_id
+    # - Filter any API-specific data
+    # - Filter any platform-specific data
     # - Inherit data from the given base_hash (if any) and extend/override it
     #   with the given input data.
     # - Append 'config_set_append' data to any existing 'config_set' data
     # - Append 'config_get_token_append' data to 'config_get_token', ditto
-    def self.hash_merge(input_hash, api, product_id, base_hash=nil)
```

Comment:
Did it so that I don't have to explicitly pass instance.api and instance.product_id as arguments.

However note that in the YAML refactor branch this is different again - this work should be considered as a work-in-progress to get unblocked with but I'll have to sort things out once the YAML enhancements are merged.